angular-workspace/projects/example-client-app/src/app/app.module.ts

```diff
@@ -7,7 +7,7 @@ import { NimbleTextAreaModule, NimbleTextFieldModule, NimbleNumberFieldModule, N
     NimbleTabModule, NimbleTabPanelModule, NimbleTabsModule, NimbleTabsToolbarModule, NimbleMenuModule,
```

Comment:
nit: just doing a text search, would be nice to clean-up variables and test names to use radio consistently instead of radiobutton

![image](https://user-images.githubusercontent.com/1588923/194598633-a7df1f28-2a0b-4e08-a89c-618f24eb38ae.png)

Reply:
Changed